content/integrate/redis-data-integration/release-notes/rdi-1-4-0.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: Redis Data Integration release notes 1.4.0 (October 2024)
+alwaysopen: false
+categories:
+- docs
+- operate
+- rs
+description: Installation on Kubernetes with a Helm chart. Improvements for installation on VMs.
+linkTitle: 1.4.0 (October 2024)
+toc: 'true'
+weight: 997
+---
+
+> This is a GA version of Redis Data Integration (RDI) that improves the installation of RDI.
+
+RDI’s mission is to help Redis customers sync Redis Enterprise with live data from their slow disk-based databases to:
+
+- Meet the required speed and scale of read queries and provide an excellent and predictable user experience.
+- Save resources and time when building pipelines and coding data transformations.
+- Reduce the total cost of ownership by saving money on expensive database read replicas.
+
+RDI keeps the Redis cache up to date with changes in the primary database, using a [_Change Data Capture (CDC)_](https://en.wikipedia.org/wiki/Change_data_capture) mechanism.
+It also lets you _transform_ the data from relational tables into convenient and fast data structures that match your app's requirements. You specify the transformations using a configuration system, so no coding is required.
+
+## Headlines
+
+-   Installation on [Kubernetes]({{< relref "/integrate/redis-data-integration/installation/install-k8s" >}}) using a [Helm chart](https://helm.sh/docs/). You can install on [OpenShift](https://docs.openshift.com/) or other flavours of K8s using Helm.
+
+- Improvements for installation on VMs:
+	- Installer checks if the OS firewall is enabled on Ubuntu and RHEL.
+	- Installer verifies DNS resolution from RDI components.
+	- Installer provides log lines from components that failed during RDI deployment if a problem occurs.
+	- Improved verification of RDI installation.
+	- Installer verifies if the RDI database is in use by another instance of RDI.
+	- Installer checks and warns if any [`iptables`](https://www.netfilter.org/projects/iptables/index.html) rules are set.
+	- Improved message when RDI tries to connect to its Redis database with invalid TLS keys.
```

Comment:
what is it linked to?

Reply:
Links to the iptable home page. Again, I'll remove it if you think it's better without it.

Reply:
the link needs to be just on the word IPtables